### PR TITLE
golang sdk 0.1.21 rns added

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ golang sdk release notes added. 
* Jira/GitHub Issue numbers (if any): \   FFM--11297
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes: version released to at least one prod environment.

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -37,6 +37,16 @@ Security Updates:
 
 ### Golang SDK
 
+#### Version 0.1.21
+
+*Enhancements*:
+ - SDK support for processing `AND/OR` rules:
+  -- note that this feature is not enabled yet and the SDK will continue to use existing group rule format until further notice. (FFM-11297)
```
Comment (douglas-j-bothwell) [inline release-notes/feature-flags.md:44]:
```suggestion
  -- Note that this feature is not enabled yet and the SDK will continue to use existing group rule format until further notice. (FFM-11297)
```

**Review by douglas-j-bothwell — APPROVED** [review]
One tweak, otherwise LGTM

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Feature Flags release notes
 sidebar_label: Feature Flags
-date: 2024-04-17T16:19:25
+date: 2024-04-23T16:19:25
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:4]:
```suggestion
date: 2024-04-24T16:19:25
```

File: release-notes/feature-flags.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Review the notes below for details about recent changes to Harness Feature Flags
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-### Latest Updated: April 17, 2024
+### Latest Updated: April 23, 2024
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:18]:
```suggestion
### Last updated: April 24, 2024
```

File: release-notes/feature-flags.md [hunk 4]
```diff
@@ -37,6 +37,16 @@ Security Updates:
 
 ### Golang SDK
 
+#### Version 0.1.21
+
+*Enhancements*:
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:42]: Though these aren't headings for your module's RN format, you might consider using the established standard headings:

New features and enhancements
Fixed issues
Early access features
Deprecations and removals

https://harness.atlassian.net/wiki/spaces/DOCS/pages/1689845830/Write+release+notes#Format-and-content-of-release-notes

File: release-notes/feature-flags.md [hunk 5]
```diff
@@ -37,6 +37,16 @@ Security Updates:
...
+ - SDK support for processing `AND/OR` rules:
+  -- Note that this feature is not enabled yet and the SDK will continue to use existing group rule format until further notice. (FFM-11297)
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:44]:
```suggestion
  - Note that this feature is not enabled yet and the SDK will continue to use the existing group rule format until further notice. (FFM-11297)
```

Comment (brian-f-harness) [inline release-notes/feature-flags.md:44]: Not sure if this format was intentional or meant to be a sub-bullet. In the preview, there are two dashes.

File: release-notes/feature-flags.md [hunk 6]
```diff
@@ -37,6 +37,16 @@ Security Updates:
...
+ - Internal metrics code enhancements.
+
+*Bug Fixes*:
+ - Fixes an issue where only a single target would be registered in a metrics window.(FFM-11297)
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:48]:
```suggestion
 - Fixed an issue where only a single target would be registered in a metrics window.(FFM-11297)
```

**Review by brian-f-harness — DISMISSED** [review]
Looks good overall. Recommend updating dates. Also added a few suggestions for you to consider.

File: release-notes/feature-flags.md (hunk 3 above)
Comment (brian-f-harness) [reply release-notes/feature-flags.md:18]: Heading level 3 shows in the mini toc on the right side of the page. Might consider making this a level 4.

File: release-notes/feature-flags.md (hunk 5 above)
Comment (jenniferopal) [reply release-notes/feature-flags.md]: That was a typo. I corrected. Thank you. :)

**Review by HarnessTheChaos — APPROVED** [review]
This looks aligned to the ticket work complete. Thanks!

File: release-notes/feature-flags.md [hunk 7]
```diff
@@ -37,6 +37,16 @@ Security Updates:
 
 ### Golang SDK
 
+#### Version 0.1.21
+
+*New Features and Enhancements*:
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:42]:
```suggestion
*New features and enhancements*:
```

File: release-notes/feature-flags.md [hunk 8]
```diff
@@ -37,6 +37,16 @@ Security Updates:
...
+ - SDK support for processing `AND/OR` rules:
+  - Note that this feature is not enabled yet and the SDK will continue to use the existing group rule format until further notice. (FFM-11297)
+ - Internal metrics code enhancements.
+
+*Fixed Issues*:
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:47]:
```suggestion
*Fixed issues*:
```

File: release-notes/feature-flags.md [hunk 9]
```diff
@@ -37,6 +37,16 @@ Security Updates:
 
 ### Golang SDK
 
+#### Version 0.1.21
+
+*New Features and Enhancements*:
+ - SDK support for processing `AND/OR` rules:
+  - Note that this feature is not enabled yet and the SDK will continue to use the existing group rule format until further notice. (FFM-11297)
+ - Internal metrics code enhancements.
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:45]: Is there a Jira associated with the enhancements?

**Review by brian-f-harness — APPROVED** [review]
Approved. If you add a commit to address the comments, you can merge without re-review.

File: release-notes/feature-flags.md (hunk 9 above)
Comment (jenniferopal) [reply release-notes/feature-flags.md:45]: None that I could find.

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**